### DISCUSSION TITLE
Position reset

### DISCUSSION THREAD

**Discussion by efb4f5ff-1298-471a-8973-3d47447115dc** [community/community#35308] [Mobile]
Clicking on issue -> going back to backlog -> results in position reset which makes me need to scroll all the way back to where i was

https://user-images.githubusercontent.com/73130443/194250170-02eb22d8-b060-4bdf-bfff-d11790e242da.mp4

**Comment by HenSquared** [comment]
Hi @efb4f5ff-1298-471a-8973-3d47447115dc 

Thanks for reporting this issue. The repro steps and the video are helpful! 

I can see how this can be frustrating. I've created an issue internally so our team can take a look at this. 

Thanks :)

**Reply by homersimpsons** [reply]
Also this does not just apply to issue list, opened files should remember scroll position. In fact I often read documentation or such (mainly markdowns) on GitHub mobile and each time I click a link (to another file) I then have to scroll back to where I was.
